cdap-ui/app/hydrator/templates/partial/node-config-modal/node-config/plugin-edit-properties-template.html

```diff
@@ -35,6 +35,9 @@
     <div class="widget-group-container">
       <div class="field-header subtitle">
         <h4>{{::group.display}}</h4>
+        <h4>
```

Comment:
should we add an ng-if here as well? since otherwise the empty description will add some padding between labels and the first widgets

Reply:
The `.field-header .subtitle` is display flex which makes sure the heading and the tooltip icon appear on the same line. I don't believe they will cause any spacing issue between heading and the first widget. Do you see a scenario where this might not be the case?